lib/iex/resources/company.rb

```diff
@@ -11,10 +13,10 @@ class Company < Resource
       property 'issue_type', from: 'issueType' # common issue type of the stock
       property 'sector'
 
-      def self.get(symbol)
-        new IEX::Api::Company.get(symbol)
-      rescue Faraday::ResourceNotFound => e
-        raise IEX::Errors::SymbolNotFoundError.new(symbol, e.response[:body])
+      def self.get(stock_symbol)
```

Comment:
Decided to rename our parameter to `stock_symbol` instead to differentiate from  `Base.symbol` method name.